htlcswitch/link_test.go

```diff
@@ -3725,7 +3725,6 @@ func (h *persistentLinkHarness) restart(restartSwitch bool,
 
 	// First, remove the link from the switch.
 	h.coreLink.cfg.Switch.RemoveLink(h.link.ChanID())
-	h.coreLink.WaitForShutdown()
```

Comment:
Rational behind removing this?

Reply:
`RemoveLink` inherently blocks until shutdown

Reply:
Its `godoc` should be updated to note this.

Reply:
fixed!